### PR TITLE
Python wrapper for Mela class

### PR DESCRIPTION
**Pull request by hroskes**

### CODE REVIEW

**Comment by usarica** [comment]
Can you post the warning message?

**Comment by hroskes** [comment]
https://github.com/usarica/HiggsAnalysis-ZZMatrixElement/pull/7/files#diff-9a1de3d9d1f33ad348d7b1ad86529dcbR57

**Comment by hroskes** [comment]
I'll keep this one open as I update the python wrapper, and keep it up to date with your newVH.

File: setup.sh [hunk 1]
```diff
@@ -1,8 +1,10 @@
 #!/bin/sh
 
-
-#echo "$@"
+(
+cd $(dirname ${BASH_SOURCE[0]})
```
Comment (usarica) [inline setup.sh:4]: Needs fixing:

[usarica@lxplus013 MELA]$ cd $(dirname ${BASH_SOURCE[0]})
dirname: missing operand
Try `dirname --help' for more information.

Comment (hroskes) [reply setup.sh:4]: ${BASH_SOURCE[0]} only works in a script.  It gives the name of the script you are calling (whether from ./setup.sh or source ./setup.sh).

**Comment by usarica** [comment]
Please re-open the PR once other items are final, and remove CMSSW dependencies

**Pull request closed without merging**